src/lib/__tests__/server-utilities.test.js

```diff
@@ -38,16 +40,31 @@ describe('md5Hash', () => {
 
 describe('compressJPEG', () => {
   if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
-  fs.closeSync(fs.openSync('./tmp/test.jpg', 'a')); // Create empty file 'test.jpg'
   const filePath = './tmp/test.jpg';
 
-  it('overwrites uncompressed image with compressed image', () => {
+  // Copy compressible JPEG to `./tmp` directory to mimick JPEG upload
+  copyFile('./static/test-assets/compression-test.jpg', filePath);
+
+  /* Not all files will compress */
```

Comment:
Maybe just elaborate with 2-3 more words that show that the file we chose will, but that some won't.